sdk/typescript/src/transactions/Commands.ts

```diff
@@ -46,13 +46,13 @@ export const Commands = {
 					typeArguments?: string[];
 			  },
 	): TransactionShape<'MoveCall'> {
-		const [pkg, mod, fn] =
+		const [pkg, mod = '', fn = ''] =
 			'target' in input ? input.target.split('::') : [input.package, input.module, input.function];
 
 		return {
 			$kind: 'MoveCall',
 			MoveCall: {
-				package: normalizeSuiObjectId(pkg),
+				package: pkg,
```

Comment:
This is a change @mamos-mysten requested

Reply:
Curious if that will work with `0x2` or any other short-cutted system addresses after this change? (or if we need to normalize on the serialization step?)

(Also I love the manos/manos tags 😆 )

Reply:
Yes, short hand package addresses should still work, we still normalize later